LambdaEngine/Include/Input/API/ActionCodes.h

```diff
@@ -30,14 +30,17 @@ namespace LambdaEngine
 		// General
 		ACTION_GENERAL_SCOREBOARD = 12,
 
+		//HUD
+		ACTION_ESCAPE			= 13,
```

Comment:
Naming for the previous enums follow a structure of ACTION_[TYPE]_[NAME]. This enum does not contain the [TYPE] which should be GENERAL considering the position of it in the list.